core/docker/build-remote.sh

```diff
@@ -19,7 +19,8 @@ WORK_DIR="$(mktemp -d)"
 curl -o ${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz ${SERVER_LOCATION}
 tar -C ${WORK_DIR} -xzf ${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz
 rm ${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz
-cp -R bin default ${WORK_DIR}/trino-server-${TRINO_VERSION}
+cp -R bin ${WORK_DIR}/trino-server-${TRINO_VERSION}
+cp -R default -t ${WORK_DIR}
```

Comment:
hmm - why so?

Reply:
Previously the default was applied dynamically in startup script 
https://github.com/trinodb/trino/blob/297ba4d58a526667d91b9685b453faafa869c68f/core/docker/bin/run-trino#L9

now it's statically applied just by the fact it's in the expected place in the image.

Reply:
Oh yeah. I see where it is referenced now.